src/nvim/misc2.c

```diff
@@ -459,22 +461,33 @@ char *read_string(FILE *fd, size_t cnt)
   return (char *)str;
 }
 
-/// Write a number to file "fd", MSB first, in "len" bytes.
-/// @return OK/FAIL.
-int put_bytes(FILE *fd, uintmax_t number, unsigned int len)
+/// Writes a number to file "fd", MSB first, in "len" bytes.
+/// @returns false in case of an error.
```

Comment:
Isn't `@return` the convential tag for the return value? I didn't know that the `@returns` alias existed, but I see you've used it throughout.